packages/react/src/alert-dialog/popup/AlertDialogPopup.tsx

```diff
@@ -1,5 +1,6 @@
 'use client';
 import * as React from 'react';
+import { useForkRef, InteractionType, inertValue } from '@base-ui-components/utils';
```

Comment:
We've been moving towards non-barrel imports in others parts of the codebase, I think for DX reasons. Might not matter as much for a small package like the utils, but it would be nice to agree on a direction.

Reply:
But during bundling (on the end-users side), the paths won't be part of the final output. It only saves code in the source code and is immaterial to the built artifacts.

Reply:
But wouldn't those imports also exist in the barrel file and therefore pulled into the bundle anyway? I'm not sure whether they fully resolve imports or include a chunk for the barrel file, but in the latter case you're definitely getting larger bundles, in the former it'd be the same. As far as I'm aware, bundlers will also rewrite those specifiers so the length doesn't influence the final bundle size.